src/jit/morph.cpp

```diff
@@ -14057,6 +14057,12 @@ GenTree* Compiler::fgMorphModToSubMulDiv(GenTreeOp* tree)
         denominator = fgMakeMultiUse(&tree->gtOp2);
     }
 
+    if (lvaLocalVarRefCounted)
```

Comment:
Right, that's my point - we catch the places where we fail to increment, but we don't catch the places where we over-increment.

---
In reply to: [94659105](https://github.com/dotnet/coreclr/pull/8804#discussion_r94659105) [](ancestors = 94659105)